PDFUnblocker.py

```
@@ -0,0 +1,74 @@
+from tqdm import tqdm
+
+# DEFINE FUNCTIONS HERE
+
+def SingleUnlock(filename: str) -> None:
+    try:
+        with pikepdf.open(filename,
+                        allow_overwriting_input=True) as pdf:
+            pdf.save(filename)
+    except pikepdf.PasswordError:
+        print(f"The file {filename} appears to be unable to be de-encrypted.")
+    except Exception as e:
```

Ruff

Do not catch blind exception: `Exception`

(BLE001)